### ISSUE TITLE
Failed retrieving config from controller (nodejs)

### ISSUE THREAD

**Issue by haf** [deis/deis#3290]
I'm trying to publish a tiny NodeJS app - just for a static web site, and I'm getting this error:

```
2015/03/13 12:51:43 failed retrieving config from controller: &{%!s(*http.body=&{0xc20800ae70 0xc208058090 0xc208062420 true {0 0} false}) {%!s(int32=0) %!s(uint32=0)} %!s(bool=false) <nil> %!s(func(error)=0x49c180) %!s(func() error=0x49c110)
```

When pushing [webpack/react-start](https://github.com/webpack/react-starter) as-is. I don't really understand the error message.

Latest released Deis.

**Comment by aledbf** [comment]
@haf this is fixed in #3137

**Comment by aledbf** [comment]
@haf check the controller logs to see the cause of the error running `journalctl -u deis-controller -n 100` in the node where `deis-controller` is running
